tests/baselines/reference/interfaceWithMultipleBaseTypes.errors.txt

```diff
@@ -1,20 +1,14 @@
 tests/cases/conformance/interfaces/interfaceDeclarations/interfaceWithMultipleBaseTypes.ts(21,11): error TS2430: Interface 'Derived2' incorrectly extends interface 'Base2'.
-  Types of property 'x' are incompatible.
-    Type '{ a: string; b: number; }' is not assignable to type '{ b: string; }'.
-      Types of property 'b' are incompatible.
-        Type 'number' is not assignable to type 'string'.
+  The types of 'x.b' are incompatible between these types.
+    Type 'number' is not assignable to type 'string'.

```

Comment:
Does `The types of 'x.b' are incompatible between these types.` feels grammatically odd to you? 
Maybe `The types of 'x.b' are incompatible`.

Though I guess it could be because it's going to mention it on the next line: in which case:

`The types of 'x.b' are incompatible between these types:` maybe?

Reply:
The period vs colon argument could be had for every line of our pyramid of elaborations - we've settled on period right now 🤷‍♂